includes/functions.php

```diff
@@ -3987,7 +3987,82 @@ function pmpro_get_ip() {
 }
 
 /**
- * Send the WP new user notification email, but also check our filter.
+ * Determines if this order can be refunded
+ * @param  object $order The order that we want to refund
+ * @return bool Returns a bool value based on if the order can be refunded
+ */
+function pmpro_allowed_refunds( $order ) { 
+
+	if( empty( $order ) || empty( $order->gateway ) || empty( $order->status ) ) {
```

Comment:
```suggestion
	if( empty( $order ) || empty( $order->gateway ) || empty( $order->status ) || empty( $order->payment_transaction_id ) ) {
```

Check for order transaction ID (props @mircobabini for the catch)